### PR TITLE
Fix layout resizing

### LINKED ISSUE #11: App layout changes after clicking on banner ad
After clicking on banner ad and then going back to app and open keyboard, Keyboard doesn't open over the app rather it opens below it shrinking and pushing app above.
This completely changes the layout of my app which is ofcourse not desired.

> Steps to reproduce:
1. Load Demo Project
2. Add a line edit node
3. Turn off immersive mode during export
4. Click on banner ad in app
5. Go back to the app
6. Click on line edit to open keyword
App will we pushed above when keyword shows up
Before clicking 

After Clicking

### PR DESCRIPTION
**Pull request by syntaxerror247**
fixes #11

### CODE REVIEW

**Comment by cengiz-pz** [comment]
Thanks for your PR. I will brainstorm over this one to see if there is a more suitable solution.

**Comment by syntaxerror247** [comment]
Hi @cengiz-pz  I have updated this PR with a better solution. 

I think issue was Admob was setting `SoftInputMode` to `adjustResize`. This PR is setting it to `adjustNothing` which is default and used by godot (it is not explicitly set in godot app, so it must be default)

I have tested it, but It would be better to test it locally for yourself too 🙂

File: admob/src/main/java/org/godotengine/plugin/android/admob/AdmobPlugin.java [hunk 1]
```diff
@@ -792,6 +793,7 @@ public void reset_consent_info() {
 	@Override
 	public View onMainCreate(Activity activity) {
 		this.activity = activity;
+		this.activity.getWindow().setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_ADJUST_NOTHING);
```
Comment (cengiz-pz) [inline admob/src/main/java/org/godotengine/plugin/android/admob/AdmobPlugin.java:796]: LGTM.

**Review by cengiz-pz — APPROVED** [review]
LGTM.

**Comment by cengiz-pz** [comment]
Tested on Samsung.

**Pull request merged**